test/cucumber/features/cluster-mode/hapi-fhir.cluster.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Fhir Datastore HAPI-FHIR?
+    Does the Fhir Datastore HAPI-FHIR package work as expected
+
+    Scenario: Init Fhir Datastore HAPI-FHIR
+        Given I use parameters "init fhir-datastore-hapi-fhir --dev --env-file=.env.cluster"
+        When I launch the platform with params
+        Then The service "postgres-1" should be started with 1 replicas
```

Comment:
```suggestion
        Then The service "postgres-01" should be started with 1 replicas
```

Reply:
This update wasn't merged yet and we will not merge it till we deploy the backup strategy for Ethiopia on platform, so for now it will be just postgres-1, mongo-1, ... till that happens